package.json

```diff
@@ -27,6 +27,7 @@
 		"@sveltejs/adapter-auto": "^3.2.2",
 		"@sveltejs/adapter-node": "^5.2.0",
 		"@sveltejs/adapter-static": "^3.0.2",
+		"@sveltejs/adapter-vercel": "^5.4.1",
```

Comment:
This should not need to be added. Adapter-auto does the trick

Reply:
Ok, i'm goonna remove e'm